Source/WebCore/rendering/RenderObject.h

```diff
@@ -1206,7 +1223,8 @@ class RenderObject : public CachedImageClient, public CanMakeCheckedPtr {
     CheckedRef<Node> m_node;
 
     SingleThreadWeakPtr<RenderElement> m_parent;
-    SingleThreadWeakPtr<RenderObject> m_previous;
+    SingleThreadPackedWeakPtr<RenderObject> m_previous;
```

Comment:
Did you mean to make this change in this PR? It isn't mentioned in the commit message.

Reply:
Yes, this is intentional as we'd increase the size of RenderObject otherwise.